accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceinstaller/AceBeanInstallerIncremental.java

```diff
@@ -134,6 +141,18 @@ protected void installAcl(Set<AceBean> aceBeanSetFromConfig, String path, Set<St
 
         }
 
+        // will override the logic to apply all ACLs on the path, and removing all elements present in config files

```

Comment:
wouldn't it make more sense that instead of reapplying everything just reorder the deny?